### DISCUSSION TITLE
Add a method to get the options collection in interaction.options.

### DISCUSSION THREAD

**Discussion by PeterStark000** [discordjs/discord.js#6155] [Ideas]
Since `interactions.options` is no longer a collection, some of my commands have broken, like the `/poll` command which has the function of creating polls.

So I ask that some method be created like `options.cache` which returns a `Collection<String, CommandInteractionOption>`.

My code ended up crashing:
![image](https://user-images.githubusercontent.com/40671520/126356269-5f30f41b-0868-4409-a6d9-3c815ca7836c.png)

How my command worked: 
![image](https://user-images.githubusercontent.com/40671520/126356572-231ce07c-6801-4180-b1aa-206b624e00d6.png)

**Comment by DTrombett** [comment]
This was asked by a lot of users and I think it'll be implemented something like this (not with a `.cache` property since it's for managers) but for now if you want a temporary solution you can use the **private** property [`._options`](https://discord.js.org/#/docs/main/master/class/CommandInteractionOptionResolver?scrollTo=_options), which is an array of [`CommandInteractionOption`](https://discord.js.org/#/docs/main/master/typedef/CommandInteractionOption).

Edit: #6156 was opened!

**Reply by PeterStark000** [reply]
Thanks for the answer
